crates/matrix-sdk-ui/src/timeline/controller/aggregations.rs

```diff
@@ -642,11 +642,7 @@ fn resolve_edits(
         }
     }
 
-    if let Some(edit) = best_edit {
-        edit_item(event, edit)
-    } else {
-        false
-    }
+    if let Some(edit) = best_edit { edit_item(event, edit) } else { false }
```

Comment:
Ending a function in `if let` feels super strange to me, I think I liked the previous version better.